packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -18,41 +18,71 @@ import { Link } from '@reach/router';
 
 ### Variations
 
+Each button variant has a particular function and should be implemented consistently across products.
+
+
 ```ts file="./ButtonVariations.tsx"
 ```
 
+| Variation | Description|
+| --- | ---|
+| Primary | A primary button is the most prominent button on a page, used for the most important call to action on a page.|
+| Secondary | Secondary buttons are buttons with less visual prominence than primary buttons. Use secondary buttons for general actions on a page, that don’t require as much emphasis as primary button actions. |
+| Danger Secondary | If secondary buttons include destructive actions, use a dangery secondary button to indicate this risk.|
+| Tertiary | Tertiary buttons are one the least visually prominent buttons and are designed to be less striking than a primary or secondary button, while still retaining a classic button format.  |
+| Danger | Danger buttons are the most prominent of all the button options. Use danger buttons for actions a user can take that are potentially destructive or difficult/impossible to undo, like deleting or removing user data. |
+| Warning | Warning buttons should be used when an action may chage an important setting or behavior, but not in a destructive or irreversible way. |
+| Link | Links buttons are labeled buttons with no background or border. Link buttons can be used for actions that may not need as much emphasis, navigate users to another page within the same window, and/or avigate users to an external page. |
+| External Link | ?|
+| Inline Link | Inline text can be used as a link button.|
+| Danger Link | Danger link are text based, but convey the same urgency as danger buttons.|
+
 ### Disabled
 
+Disable buttons when the action cannot be completed. Every button variant can be disabled.
+
 ```ts file="./ButtonDisabled.tsx"
 ```
 
 ### Aria disabled
 
+Aria disabled buttons are similar to normal disabled buttons, but can be focused so that a user may select the button using a keyboard. Every button variant can be aria disabled.
```

Comment:
For this, I'm wondering if "a user may select the button using keyboard" may come off as meaning the button can still be clicked and function. This could also just be me, but just to suggest a tweak on this description:

```suggestion
Buttons that are `aria-disabled` are similar to normal disabled buttons, except they can receive focus. Every button variant can be aria disabled.
```
Mainly included the `aria-disabled` attribute in backticks to signify it's an attribute name, removed the line about "select the button...", and also removed the mention of keyboard since focus can also be placed via mouse (though this might depend if we want to convey only visual focus here).